regulations/static/regulations/css/scss/_typography.scss

```diff
@@ -178,13 +178,13 @@ a:visited {
 .internal:visited {
   color: $black;
   text-decoration: none;
-  border-bottom: 1px solid $blue_light;
+  border-bottom: 1px solid $action_color;
 }
 
 .internal:hover,
 .internal:active,
 .internal:focus {
-  background-color: $blue_lighter;
+  background-color: $action_color_dark;
```

Comment:
This doesn't seem quite right -- won't this be a much darker color than what it was before?